dask_expr/reductions.py

```diff
@@ -180,52 +356,72 @@ def chunk_kwargs(self):
             min_count=self.min_count,
         )
 
-    @property
-    def _meta(self):
-        return self.frame._meta.sum(**self.chunk_kwargs)
-
     def _simplify_up(self, parent):
         if isinstance(parent, Projection):
             return self.frame[parent.operand("columns")].sum(*self.operands[1:])
 
 
-class Max(Reduction):
-    _parameters = ["frame", "skipna"]
+class Sum(_Sum, DataFrameReduction):
+    map = _Sum.make_map()
+    reduce = _Sum.make_reduce()
+
+
+class _Max(DataFrameReductionTemplate):
+    _parameters = ["frame", "skipna", "numeric_only"]
+    _defaults = {"skipna": True, "numeric_only": None}
```

Comment:
This looks like it's making things really complex?  For every reduction we now need to define two classes?